app/views/saasbook/book_content.html.erb

```diff
@@ -39,5 +40,6 @@
                 <%= render @body_contents %>
             </div>
         </div>
+        <%= content_tag :div, class: "page_information", data: {chapter: @chapter_id, section: @section_id} do %><% end %>

```

Comment:
whats the content tag for? you should be able to access the variables from anywhere with ruby, no?

Reply:
Please read the PR description for context on what this tag is for. Ruby can access the variables, but Javascript cannot. This is the cleanest way (unfortunately) to pass Ruby variables to Javascript. This was discussed this in class a few times, as well as at coach meetings.

The PR description included information about the inclusion of this tag:
"app/views/saasbook/book_content.html.erb: Note the tag at the bottom. This is how we are passing information from our controller to javascript!"